common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -248,19 +248,16 @@ def _hidden_content_student_view(self, context):
         display depending on whether staff is masquerading.
         """
         if not self._can_user_view_content():
-            subsection_format = (self.format or _("subsection")).lower()  # pylint: disable=no-member
+            date_reason = "course has ended" if getattr(self, 'self_paced', False) else "due date has passed"
```

Comment:
Yup - see below comment for more. Sentences are the smallest blocks acceptable for translation per [our internationalization guide](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html), so we'll need to explicitly enumerate each possibility.